collect_app/src/main/java/org/odk/collect/android/dao/CursorLoaderFactory.java

```diff
@@ -0,0 +1,153 @@
+package org.odk.collect.android.dao;
+
+import androidx.loader.content.CursorLoader;
+
+import org.odk.collect.android.application.Collect;
+import org.odk.collect.android.instances.Instance;
+import org.odk.collect.android.provider.InstanceProviderAPI;
+
+public class CursorLoaderFactory {
```

Comment:
The plan for the moment is to move all the `CursorLoader` calls here. Removing `CursorLoader` is probably going to be a job for sometime in the future so having them located in one easy to find place will hopefully make this easier.